app/routes/email.js

```diff
@@ -14,46 +14,46 @@
  * limitations under the License.
  */
 
-import Ember from "ember";
+import Ember from 'ember';
 
-var EmailRoute = Ember.Route.extend({
-  model: function(){
+const EmailRoute = Ember.Route.extend({
+  model() {
     return Ember.$.get('http://localhost:2845/email');
   },
   actions: {
-    'refreshMessageList': function() {
+    refreshMessageList() {
       this.refresh();
 
       // restart the timer
       Ember.run.cancel(this.refreshTimer);
       this.refrshTimer = Ember.run.later(this, this.refreshModel, 5000);
     },
-    'clearMessageList': function() {
-      var _this = this;
-      return new Ember.RSVP.Promise(function(resolve) {
-        Ember.$.ajax('http://localhost:2845/email', { type: 'DELETE', statusCode: { 200: function(data){ resolve(data); } } });
-      }).then(function(data){
+    clearMessageList() {
+      const _this = this;
+      return new Ember.RSVP.Promise(((resolve) => {
```

Comment:
Looks like there's an extra set of parenthesis introduced here and in line 35.

Reply:
Fixed. Thanks